block/manager.go

```diff
@@ -170,7 +170,24 @@ func (m *Manager) SetDALC(dalc da.DataAvailabilityLayerClient) {
 
 // AggregationLoop is responsible for aggregating transactions into rollup-blocks.
 func (m *Manager) AggregationLoop(ctx context.Context) {
+	initialHeight := uint64(m.genesis.InitialHeight)
+	height := m.store.Height()
+	var delay time.Duration
+
+	// TODO(tzdybal): double-check when https://github.com/celestiaorg/rollmint/issues/699 is resolved
+	if height < initialHeight {
+		delay = time.Until(m.genesis.GenesisTime)
+	} else if height >= initialHeight {
```

Comment:
do you need `if height >= initialHeight`? just `else` is enough.

Reply:
Good catch. Keep things simple :+1: